WORKSPACE

```diff
@@ -32,6 +32,9 @@ bind(
     actual = "//external:ssl",
 )
 
+# envoy commit date 04/10/2019
```

Comment:
I doubt some of the conflicts are not resolved correctly. John's PR shoudl beat this one: https://github.com/istio/proxy/commit/0b76a30145d2a67429758ebe7c34672abd9e0498

Reply:
This is the redundant message. John's PR is down. If you check line 41, you will find John's PR. But I don't know why I didn't find this line when I resolving conflicts.

Reply:
Sure. git is not that reliable :) That's why we have this review 
Please fix it in the follow up